README.md

```diff
@@ -21,7 +21,7 @@ The Controller is the central API server for [Deis Workflow][workflow]. It is in
 * Configure an application
 * Create a new user
 
-# Development
+# Contributing
```

Comment:
These clarifications to the docs are great, but should be in a different PR ideally since they're not related to ingress changes.